### PR TITLE
Adtira/redline search

### PR DESCRIPTION
**Pull request by tirada**
- Added debouncing to search experience
- Added lodash package
- Fixed minor bugs in ListView and CardView
- ~~Removed Query State from SearchBar component~~

### CODE REVIEW

File: src/components/CardView.tsx [hunk 1]
```diff
@@ -54,15 +54,19 @@ export const CardView: React.FC<IItemListProps> = (props: IItemListProps): JSX.E
           </Flex.Item>
           <Flex.Item size="size.half" grow>
             <Flex column styles={{ textAlign: 'left' }}>
-              <Flex.Item>
+              <Flex.Item
+                styles={{ display: '-webkit-box', WebkitBoxOrient: 'vertical', WebkitLineClamp: 1, overflow: 'hidden' }}
```
Comment (sachingupta) [inline src/components/CardView.tsx]: does it work in IE/edge ?

File: src/components/SearchBar.tsx [hunk 2]
```diff
@@ -31,7 +33,7 @@ export const SearchBar: React.FC<ISearchBarProps> = (props: ISearchBarProps): JS
 
   const handleKeyPress = (event: React.KeyboardEvent<HTMLInputElement>): void => {
     if (event.key === 'Enter') {
-      props.onSearch(Query);
+      props.onSearch((event as React.SyntheticEvent<HTMLInputElement>).currentTarget.value);
```
Comment (sachingupta) [inline src/components/SearchBar.tsx]: Use Query don't directly use input field value.

File: src/components/CardView.tsx (hunk 1 above)
Comment (tirada) [reply src/components/CardView.tsx]: image

Will work on Edge, but not IE. Should we support IE?

Comment (sachingupta) [reply src/components/CardView.tsx]: Create a bug to support this in IE and will followup with PM

**Pull request merged**